### PR TITLE
notcurses 2.4.1

### PR DESCRIPTION
**Pull request by dankamongmen**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/notcurses.rb [hunk 1]
```diff
@@ -21,8 +21,13 @@ class Notcurses < Formula
   depends_on "libunistring"
   depends_on "ncurses"
   depends_on "readline"
+
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "gcc"
+  end
```
Comment (carlocab) [inline Formula/notcurses.rb:29]:
```suggestion
  end

  fails_with gcc: "5"
```

**Review by carlocab — DISMISSED** [review]
Thanks, @dankamongmen.

File: Formula/notcurses.rb (hunk 1 above)
Comment (dankamongmen) [reply Formula/notcurses.rb:29]: it ought work with clang, fwiw

Comment (carlocab) [reply Formula/notcurses.rb:29]: Yea, but we normally fall back to GCC first before trying Clang. I'll just fix this in another PR.

**Comment by carlocab** [comment]
Oh, your PR branch is called `master`. Querying the GitHub API for workflows associated with branches named `master` has been broken for a while. Will open a new PR with a different branch name.

**Comment by carlocab** [comment]
Opened #85156 to replace this. 

For future reference, if you have `brew` installed and `git` authenticated,
```
brew bump-formula-pr --version=2.4.1 notcurses
```
should just do the right thing.

**Comment by dankamongmen** [comment]
> Opened #85156 to replace this.
> 
> For future reference, if you have `brew` installed and `git` authenticated,
> should just do the right thing.

attv("i have made a note of it"); thanks!

**Pull request closed without merging**